pages/common/go.md

```diff
@@ -14,6 +14,10 @@
 
 `go build`
 
+- Compile the file source.go into a named executable:
```

Comment:
Please move it to above the plain `go build` example so that we have consistency regarding running and building single files.

And use {{file}}.go for maintaining consistency.